src/main/java/org/gradle/profiler/studio/launcher/LauncherConfigurationParser.java

```diff
@@ -0,0 +1,102 @@
+package org.gradle.profiler.studio.launcher;
+
+import com.google.common.base.Joiner;
+import org.gradle.profiler.instrument.GradleInstrumentation;
+import org.gradle.profiler.studio.tools.StudioSandboxCreator.StudioSandbox;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LauncherConfigurationParser {
```

Comment:
This class was just moved and some parts of it were moved to StudioConfigurationProvider